### PR TITLE
updating bugsnag integration to match new integration flow

### PR DESCRIPTION
**Pull request by l0k0ms**
This is to address https://github.com/DataDog/documentation/issues/1231

### CODE REVIEW

File: content/integrations/bugsnag.md [hunk 1]
```diff
@@ -23,15 +23,23 @@ No installation is required.
...
 
-1. In [Bugsnag](https://bugsnag.com/) go to _Settings_ for the project you would like to configure to send notifications to Datadog
-1. Select _Team Notifications_ and then _Datadog_
-1. Customize the notifications you'll see in Datadog by selecting error notification triggers.
-1. Apply custom filters to your notification triggers to see errors from specific release stages and severities.
-1. Enter your Datadog API key.
-1. Select _Test Notification_ to test the configuration. A test error from Bugsnag will appear in Datadog's event stream.
-1. _Save_ your settings.
-1. Add more streams from the same project to see error events based on a different set of notification criteria.
+1. Go to `Settings` in [Bugsnag](https://bugsnag.com/)  for the project you would like to configure to send notifications to Datadog.
```
Comment (kmshultz) [inline content/integrations/bugsnag.md]: **Settings**, not `Settings`

File: content/integrations/bugsnag.md [hunk 2]
```diff
@@ -23,15 +23,23 @@ No installation is required.
...
 
+2. Select `Team Notifications` and then `Datadog`.
```
Comment (kmshultz) [inline content/integrations/bugsnag.md]: Again, **Team Notifications** and **Datadog**, not `Team Notifications` and `Datadog`

File: content/integrations/bugsnag.md [hunk 3]
```diff
@@ -23,15 +23,23 @@ No installation is required.
...
+
+3. Customize the notifications you'll see in Datadog by selecting error notification triggers.
+{{< img src="bugsnag_1.png" alt="bugsnag_notification_setting" >}}
+
+4. Apply custom filters to your notification triggers to see errors from specific release stages and severities.
+{{< img src="bugsnag_2.png" alt="bugsnag_filters_setting" >}}
+
+5. Enter your Datadog API key.
+
+6. Select `Test Notification` to test the configuration. A test error from Bugsnag should appear in your Datadog application.
```
Comment (kmshultz) [inline content/integrations/bugsnag.md]: **Test Notification**

**Comment by l0k0ms** [comment]
@kmshultz good for you?

File: content/integrations/bugsnag.md [hunk 4]
```diff
@@ -23,15 +23,23 @@ No installation is required.
...
 
-1. In [Bugsnag](https://bugsnag.com/) go to _Settings_ for the project you would like to configure to send notifications to Datadog
-1. Select _Team Notifications_ and then _Datadog_
-1. Customize the notifications you'll see in Datadog by selecting error notification triggers.
-1. Apply custom filters to your notification triggers to see errors from specific release stages and severities.
-1. Enter your Datadog API key.
-1. Select _Test Notification_ to test the configuration. A test error from Bugsnag will appear in Datadog's event stream.
-1. _Save_ your settings.
-1. Add more streams from the same project to see error events based on a different set of notification criteria.
+1. Go to **Settings** in [Bugsnag](https://bugsnag.com/)  for the project you would like to configure to send notifications to Datadog.
```
Comment (kmshultz) [inline content/integrations/bugsnag.md]: nit: there's one too many spaces in this line.

File: content/integrations/bugsnag.md [hunk 5]
```diff
@@ -23,15 +23,23 @@ No installation is required.
...
+
+3. Customize the notifications you'll see in Datadog by selecting error notification triggers.
+{{< img src="bugsnag_1.png" alt="bugsnag_notification_setting" >}}
+
+4. Apply custom filters to your notification triggers to see errors from specific release stages and severities.
+{{< img src="bugsnag_2.png" alt="bugsnag_filters_setting" >}}
+
+5. Enter your Datadog API key.
+
+6. Select **Test Notification** to test the configuration. A test error from Bugsnag should appear in your Datadog application.
```
Comment (kmshultz) [inline content/integrations/bugsnag.md]: This is really part of Validation, but since it's sort of part of the configuration, I guess let's leave it here.

**Pull request merged**